src/numbuf/CMakeLists.txt

```diff
@@ -58,7 +59,8 @@ add_library(numbuf SHARED
   cpp/src/numbuf/sequence.cc
   python/src/pynumbuf/numbuf.cc
   python/src/pynumbuf/adapters/numpy.cc
-  python/src/pynumbuf/adapters/python.cc)
+  python/src/pynumbuf/adapters/python.cc
+  ${COMMON_EXTENSION})
```

Comment:
This is to avoid code duplication for `PyStringToUniqueID`? Or some other reason?